### DISCUSSION TITLE
ethereum-provider.js is not define EthereumProvider

### DISCUSSION THREAD

**Discussion by 4e5ung** [WalletConnect/walletconnect-monorepo#3535] [Developer Support] [answered]
Trying to use ethereum-provider only through html.

<script src="
https://cdn.jsdelivr.net/npm/@walletconnect/ethereum-provider@2.10.0/dist/index.umd.min.js
"></script>

I am trying to import EthereumProvider through this code.
EthereumProvider is not declared.

As far as I know, it is exported on the node module.
import { EthereumProvider } from '@walletconnect/ethereum-provider'

In the case of walletconnectV1, WalletConnectProvider was imported and used.

<script src="https://cdn.jsdelivr.net/npm/@walletconnect/web3-provider@1.7.1/dist/umd/index.min.js"></script>

Please suggest a solution.

**Comment by dreamd33** [comment]
I'm encountering the same issue, have you fixed it?

**Reply by glitch-txs** [reply, marked as answer]
hi, so far for CDN on ethereum provider we got [this example](https://github.com/glitch-txs/ethereum-provider-cdn)

**Reply by 4e5ung** [reply]
It's resolved and I'm using it. Sorry for the delayed response.

**Reply by GrayFieth** [reply]
Merge
